### DISCUSSION TITLE
Configurable default layout for full-page components

### DISCUSSION THREAD

**Discussion by adamthehutt** [livewire/livewire#2221] [4. General]
Currently Livewire uses the "layouts.app" view as the default layout for full-page components. This can be overridden in the render() method of the component, but it would be a minor convenience to be able to configure an alternative default.

I have a PR ready to go for this if you think it's worth having:
https://github.com/livewire/livewire/compare/master...adamthehutt:master

**Comment by joshhanley** [comment]
@adamthehutt I think thats a great idea! Nice and simple.

Check out this issue raised by Caleb regarding layouts #2000. 

Can I suggest adding your comments and ideas there?

**Reply by adamthehutt** [reply]
Thanks. Done!
